TwilioVerifySDK/TwilioVerify/Sources/Data/KeyStorage.swift

```diff
@@ -77,6 +82,6 @@ extension KeyStorageAdapter: KeyStorage {
 
 private extension KeyStorageAdapter {
   func signerTemplate(withAlias alias: String, shouldExist: Bool = true) throws -> SignerTemplate {
-    return try ECP256SignerTemplate(withAlias: alias, shouldExist: shouldExist)
+    return try ECP256SignerTemplate(withAlias: alias, shouldExist: shouldExist, accessGroup: accessGroup)
```

Comment:
As the access group is a general configuration, we can keep it in `KeyStorage` class when creating and getting the signer instead of adding it to the template. As I see it, the template will have fields to change per key pair, but if we use the same value for all the key pairs (e.g. access group), we should include it in the key storage, same as we doing for factors and other information storage